detox/test/e2e/02.matchers.test.js

```diff
@@ -19,6 +19,11 @@ describe('Matchers', () => {
     await element(by.text('Index')).atIndex(index).tap();
     await expect(element(by.text('First button pressed!!!'))).toBeVisible();
   });
+  
+  it('should be able to swipe elements matched by index', async () => {
+    const index = device.getPlatform() === 'ios' ? 2 : 0;
```

Comment:
@LeoNatan, why 2 for iOS? Is it done on purpose, so that tests don't fail on Android?

Reply:
Yes, the order is different between iOS and Android. That's why the `atIndex()` API is terrible. There is no control over order.